generator/src/generator/namespace.rs

```diff
@@ -3572,7 +3668,16 @@ impl<'ns, 'c> NamespaceGenerator<'ns, 'c> {
                 _ => unreachable!(),
             };
             if !self.enum_has_repeated_values(&enum_def) {
-                outln!(out, "let {} = {}.try_into()?;", var_name, var_name);
+                let type_type = type_.type_.def.get().unwrap();
+                match type_type {
+                    xcbdefs::TypeRef::BuiltIn(xcbdefs::BuiltInType::Bool) => {
+                        // Cast this to a u8 before calling try_into, because there's
+                        // no TryFrom<bool> implementation.
```

Comment:
Should we add one? :-)
This seems to be about xinput's `GrabOwner` enum. That enum already has `impl From<GrabOwner> for bool`, so the code responsible for generating that can most likely easily be extended to also generate `impl From<bool> for GrabOwner`.

Reply:
Yes, this is about GrabOwner. At first I wanted to implement a bool conversion but the issue is that there's nothing in the enum_def to distinguish GrabOwner which is treated as boolean value from every other two variant enum which is not. And some of the others are decidedly not boolean in nature (e.g. sync::ValueType or xinput::ValuatorStateMask). So I didn't feel that a general boolean conversion made sense.

Reply:
Okay. Thanks.